ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2022.3.18f1
-m_EditorVersionWithRevision: 2022.3.18f1 (d29bea25151d)
+m_EditorVersion: 2022.3.34f1
```

Comment:
is this a new version? or was this branch on an old version for some reason?

Reply:
In the UI/UX branch the project got downgraded, here I revert it to the version that we currently all use